webapp/src/pages/Settings.tsx

```diff
@@ -637,13 +647,37 @@ const Settings: React.FC = () => {
   );
 
   return (
-    <Box height="100%" display="flex" flexDirection="column">
-      <Paper
-        variant="outlined"
+    <>
+      <DialogTitle id="config-dialog-title">
+        <Box display="flex" justifyContent="space-between">
+          <Typography variant="subtitle1">
+            View and edit certain fields from your config file. Once your
+            changes are saved, expect some delays for recomputing the affected
+            tasks.
+          </Typography>
+          <IconButton
+            size="small"
+            color="primary"
+            sx={{ padding: 0 }}
```

Comment:
Because of the `padding: 0`, the "touch ripple" effect on the button is not round anymore. I would remove the `sx={{ padding: 0 }}` to leave the default look of the button and add `alignItems="center"` on the flex `Box` to center the title on the button. Does that make sense?
```suggestion
        <Box display="flex" justifyContent="space-between" alignItems="center">
          <Typography variant="subtitle1">
            View and edit certain fields from your config file. Once your
            changes are saved, expect some delays for recomputing the affected
            tasks.
          </Typography>
          <IconButton
            size="small"
            color="primary"
```

Reply:
Okay, Understood. It's a nice trick. Thanks.